create/mysql.go

```diff
@@ -0,0 +1,94 @@
+package create
+
+import (
+	"context"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+
+	runtimev1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	infra "github.com/ninech/apis/infrastructure/v1alpha1"
+	meta "github.com/ninech/apis/meta/v1alpha1"
+	storage "github.com/ninech/apis/storage/v1alpha1"
+	"github.com/ninech/nctl/api"
+)
+
+type mySQLCmd struct {
+	Name                 string                                 `arg:"" default:"" help:"Name of the MySQL instance. A random name is generated if omitted."`
+	Location             string                                 `default:"nine-cz41" help:"Location where the MySQL instance is created."`
+	MachineType          infra.MachineType                      `help:"MachineType defines the sizing for a particular MySQL instance." placeholder:"nine-standard-1" default:"nine-standard-1"`
```

Comment:
Do we need to define a Default or will the API already supply one if none is chosen?

Reply:
The default for the MachineType is necessary as the API doesn't supply one.

Reply:
Ok for me for now then. Even though I think we should define one on API level aswell then.